shared/common-adapters/input.js.flow

```diff
@@ -50,8 +51,8 @@ export type Props = {
 }
 
 declare export default class Input extends Component<Props> {
-  setNativeProps(props: Object): {},
-  blur: () => void,
-  focus: () => void,
-  select: () => void,
+  setNativeProps(props: Object): {};
```

Comment:
weird that this was in there. are we not linting .flow files?

Reply:
we don't. its parser was easily confused by flow so we had to turn it off. maybe it could go back on now